src/agoradatatools/process.py

```diff
@@ -56,6 +56,13 @@ def apply_custom_transformations(datasets: dict, dataset_name: str, dataset_obj:
         return transform.transform_rna_distribution_data(datasets=datasets)
     if dataset_name == "proteomics_distribution_data":
         return transform.transform_proteomics_distribution_data(datasets=datasets)
+    if (
+        dataset_name == "proteomics"
+        or dataset_name == "proteomics_tmt"
+        or dataset_name == "proteomics_srm"
+    ):
```

Comment:
NIT: This could be simplified a bit.
```
if dataset_name in ["proteomics", "proteomics_tmt", "proteomics_srm"]:
```
If you need to be very precise, or
```
if "proteomics" in dataset_name:
```
If you don't.

Reply:
Or... `dataset_name.startswith('proteomics')` :)

Reply:
Oooh nice, thank you! Will fix.